pkg/epforwarder/epforwarder.go

```diff
@@ -437,8 +437,10 @@ func (p *passthroughPipeline) Start() {
 }
 
 func (p *passthroughPipeline) Stop() {
-	p.strategy.Stop()
-	p.sender.Stop()
+	if p.strategy != nil {
+		p.strategy.Stop()
+		p.sender.Stop()
+	}
```

Comment:
Is stopping `sender` only if `strategy` is not `nil` on purpose ?

Reply:
`sender` calls `Start` only if `p.strategy` is not nil: https://github.com/DataDog/datadog-agent/pull/22069/files/5499155b7a536ba122a2fa189d7eabf9236abbc5..f775c5fdbea2f103f3b6e6aa438ea3406659ede7#diff-6569a3f8ce509bc4602e1db6ae7933bbf5079532cbad740507d6dfe7a6fba3f4R433-R435

Reply:
Ok perfect then, thanks for the reply !